src/app/pages/AccountPage/Features/SendTransaction/index.tsx

```diff
@@ -9,7 +9,15 @@ import { useEffect, useState } from 'react'
 import { useTranslation } from 'react-i18next'
 import { useDispatch, useSelector } from 'react-redux'
 
-export function SendTransaction() {
+export interface SendTransactionProps {
+  isAddressInWallet: boolean
+}
+
+export function SendTransaction(props: SendTransactionProps) {
+  if (!props.isAddressInWallet) {
+    throw new Error('SendTransaction component should only appear on your accounts')
```

Comment:
I get idea that entire app will blow up to catch an error, but I think it's better to cover such scenarios in unit tests than in a component via throwing new Error.

Reply:
I don't think tests can ensure that we devs don't move SendTransaction outside the condition:
```tsx
      {isAddressInWallet && (
        ...
          <SendTransaction
```
Maybe with integration tests of all possible user flows :shrug: